modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -8,18 +8,27 @@ This guide contains an overview of the changes in Canvas Kit v9. Please
 [reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you have
 any questions.
 
+- [Glossary](#glossary)
 - [Codemod](#codemod)
-- [Component Deprecations](#component-deprecations)
+- [Token Updates](#token-updates)
+  - [Depth](#depth)
+- [Component Removals](#component-removals)
   - [Drawer](#drawer)
   - [Layout and Column](#layout-and-column)
-  - [Stack](#stack-hstack-vstack)
+  - [Stack, HStack, and VStack](#stack-hstack-and-vstack)
 - [Component Updates](#component-updates)
-  - [Buttons](#buttons)
-	- [Toast](#toast)
+  - [Button](#button)
+  - [Toast](#toast)
 - [Utility Updates](#utility-updates)
   - [focusRing](#focusring)
-- [Token Updates](#token-updates)
-	- [Depth](#depth)
+
+## Glossary
+
+**Main:** description of Main package (@workday/canvas-kit-react) goes here
+
+**Preview:** description of Preview package (@workday/canvas-kit-preview-react) goes here
+
+**Labs:** description of Labs package (@workday/canvas-kit-labs-react) goes here
```

Comment:
ooo i like this addition!

Reply:
Thanks! Hoping this section will clear up any confusion around the use of Main/Preview/Labs throughout the guide.